server/filestore.go

```diff
@@ -545,6 +545,13 @@ func (fs *fileStore) unlockAllMsgBlocks() {
 }
 
 func (fs *fileStore) UpdateConfig(cfg *StreamConfig) error {
+	start := time.Now()
+	defer func() {
+		if took := time.Since(start); took > time.Minute {
+			fs.warn("UpdateConfig took %v", took.Round(time.Millisecond))
```

Comment:
We get asked multiple times a week when what the warnigns around time taken for internal subs mean and how it can be fixed.  Could we avoid such questions by improving these warnings?

Reply:
We maybe need a page in the docs with some of these log lines and they mean, when these appear some of the side effects would be latency doing operations on the stream so would explain timeouts during the timespan reported.